### PR TITLE
Changes to schema_linter and travis config. Fixes #1156

### LINKED ISSUE #1156: schema_linter.py should have a configurable ontology service URL
**Description**

The schema_linter script, which runs as part of travis tests, looks for the ontology service in the dev environment when checking schemas. The dev environment isn't guaranteed to be available and so the integration/stg/prod schema environments can fail to pass tests in this scenario.

**Acceptance Criteria**

- the schema_linter has a configurable ontology service URL
- the appropriate environment-specific ontology service URL is used in travis test runs of the schema_linter

### PR DESCRIPTION
**Pull request by ESapenaVentura**
### Release notes

No schema changes. Several changes to `schema_linter.py`:
- OLS API now is passed as a parameter to the linter.schema_linter() function. This allows to set up the environment from within execution of the script.
- Parser has been added with the option to give `develop`, `integration`, `staging` and `master` as the OLS API used. `master` is not exposed to the public for the OLS API so it defaults to `staging` when given.
- To avoid breaking backwards compatibility, the argument `--environment` is not required, and defaults to `staging`

### Reviews requested

- Need 1 or 2 Reviewers to approve because this is a code update.

### CODE REVIEW

**Comment by ESapenaVentura** [comment]
@rolando-ebi Is this the change you were thinking of? (Can't tag you as a reviewer for some strange reason)

**Comment by ESapenaVentura** [comment]
@HumanCellAtlas/wranglers 

tl;dr of the changes: The schema linter will now use the proper environment for checking ontologies. You can still run it locally without giving the environment (But it will default to staging instead of dev)

File: src/schema_linter.py [hunk 1]
```diff
@@ -38,14 +38,22 @@
 
 graph_restriction_attributes = ['ontologies', 'classes', 'relations', 'direct', 'include_self']
 
-OLS_API_DEFAULT = 'https://ontology.dev.data.humancellatlas.org/api'
+OLS_VALID_ENVIRONMENTS = ['dev', 'integration', 'staging']
```
Comment (rolando-ebi) [inline src/schema_linter.py]: Could do...

`OLS_ENVIRONMENT` = {
  "develop": "dev",
  "integration": "integration",
  "staging": "staging",
  "master": "staging"
}
`
and then line 328 becomes

`
environment = OLS_ENVIRONMENT[arguments.environment]
`

**Comment by ESapenaVentura** [comment]
Applied the changes that @rolando-ebi asked for. I will proceed to merge the pull request!
